pytorch_blade/bazel_build.py

```diff
@@ -118,6 +117,21 @@ def __init__(self, *args, **kwargs):
         if running_on_ci():
             self.configs += ["--config=ci_build"]
 
+        # ----------------------------------------------------------------------- #
+        # --------------------   Settings for Quantization   -------------------- #
+        # ----------------------------------------------------------------------- #
+        # This is for quantization process starts from blade_compression.
+        # blade_compression uses fx graph to do quantization, which is
+        # introduced in torch 1.8.0. So the lower bound of torch version is
+        # set to 1.8.0
+        # note: If we use backend (such as trt) to do calibration and get the
+        # quantization engine, this limitation should not be set.
+        is_enable_quantization = self.torch_major_version == 1 and \
+                                    8 <= self.torch_minor_version < 12
```

Comment:
I am surprised that the condition expression can be written in such a fashion way in python.

Reply:
Please make sure the syntax is valid :)

Reply:
in python3.6.9, the syntax seems valid:
![image](https://user-images.githubusercontent.com/19526090/181698133-7632493f-b92d-4948-8f5c-2bf1409c398e.png)